### PR TITLE
Validate http headers in CH

### PR DESCRIPTION
**Pull request by mnot**
(FYI, if you're too far into AUTH48 don't worry about merging this, although the RFC Editor might want those sections annotated, and if so, the whitespace should be removed from the beginning of the lines)

### CODE REVIEW

**Comment by yoavweiss** [comment]
Thanks! I manually marked the examples as `http-message` as part of the AUTH48 process. Might be late for the sf.json changes though

**Comment by mnot** [comment]
No worries about `sf.json` - it's just a hint file. If you still can remove the leading whitespace that would be cool, but it's not the end of the world if you don't.

**Comment by yoavweiss** [comment]
> If you still can remove the leading whitespace that would be cool, but it's not the end of the world if you don't.

Too late for that, I believe.

**Comment by reschke** [comment]
No.

<https://www.rfc-editor.org/auth48/C414> says it's waiting for author approval. By definition, it is not too late.

**Comment by yoavweiss** [comment]
OK. I sent out the revised XML yesterday. Should I simply send another revised version that also includes these changes?

**Comment by mnot** [comment]
Yes, they'll incorporate (they're very nice that way). Please make sure you remove the leading whitespace as per this PR.

**Comment by yoavweiss** [comment]
Will do, thanks! :)

**Review by yoavweiss — APPROVED** [review]
LGTM

**Pull request merged**
